src/FirebaseFunctionsRateLimiter.ts

```diff
@@ -22,8 +22,10 @@ export class FirebaseFunctionsRateLimiter {
     public static withFirestoreBackend(
         configuration: FirebaseFunctionsRateLimiterConfiguration,
         firestore: admin.firestore.Firestore | FirestoreEquivalent,
+        createExpireAtFromMillis?: ((millis: number) => any)
```

Comment:
@nicougou not really, actually I jumped into that trap. You have to use same version of firestore-admin package to make it work. So if our project has version x, and this library has version y, it throws an error.

Reply:
@nicougou yes, but this library is using different version of firebase-admin than our project. If that happens, firebase admin throws an error because:
- we pass firebase admin from our project version 11.x
- we use Timestamp from library that has version 10.x

Then there is some validation thrown on update method from firestore.